lib/sinatra/base.rb

```diff
@@ -1936,7 +1925,7 @@ class #{self.class}
             </style>
           </head>
           <body>
-            <h2>Sinatra doesn&rsquo;t know this ditty.</h2>
+            <h2>Sinatra doesn’t know this ditty.</h2>
```

Comment:
http://practicaltypography.com/apostrophes.html says the `&rsquo;` is the correct way of representing an apostrophe. This change is converting the HTML entity to a glyph. Should be okay in most cases except if there would be a case when someone on a different machine wants to edit this and doesn't know how to input the character via the keyboard. 

`&apos;` outputs a `'` straight tick in HTML. In the image, left is `&apos;` and right is `&rsquo;`—same as the glyph `’`. 
![screenshot 2015-09-14 22 47 58](https://cloud.githubusercontent.com/assets/400299/9856495/b9fc256e-5b32-11e5-822d-3bbe727f73f6.png)